server/lib/enrich/article.js

```diff
@@ -25,6 +25,10 @@ module.exports = exports = function article(content, format) {
 		content.content_type = CONTENT_TYPE_ALIAS[content.type] || content.type;
 	}
 
+	if (content.contentStats && content.contentStats.graphics && content.contentStats.graphics > 0){
+		content.content_type = 'rich article'
```

Comment:
Better with a constant value instead of a string `rich article` 

I'm not familiar with project but it seems from the live above there's CONTENT_TYPE_ALIAS lookup table?